client/jetpack-cloud/sections/partner-portal/issue-license-v2/style.scss

```diff
@@ -25,11 +27,11 @@
 
 .issue-license-v2__actions {
 	display: flex;
-	justify-content: space-between;
 	align-items: center;
 	gap: 8px;
+}
 
-	@include breakpoint-deprecated( "<960px" ) {
-		justify-content: normal;
-	}
+.issue-license-v2__controls {
+	@include licensing-portal-bottom-action-bar;
+	padding: 0.875rem;
```

Comment:
I wanted to confirm whether this was intended or whether should we use `px` here.

Reply:
Good point. I have fixed this now: https://github.com/Automattic/wp-calypso/pull/84629/commits/47585aea46917df556454f31e8157310a09d5e0f